src/utils/nameUtils.js

```diff
@@ -1,9 +1,21 @@
+function getNameObject(individualData, context) {
+  const names = individualData?.names || [];
+  return names.find(n => n?.context === context) || {};
```

Comment:
Return an array or an object? Intentional?

Reply:
`names` is an array of name objects. `names.find` should return either the name object with the matching context or `undefined`. So, this function should return either a name object or an empty object.

Reply:
Sorry. I thought that find was like map and returned an array. Thanks!

Reply:
You might be thinking of filter.